QuickDeployment/QuickDeploymentInternetSetup.md

```diff
@@ -0,0 +1,215 @@
+    ```
+
+> :memo: **Note:**
+> Now that you've added a credential to your Nexus repositories, access to the `ChocolateyInstall.ps1` and `ClientSetup.ps1` scripts in your `choco-install` raw repository will require this credential as well. 
+
+As we will learn in the next section on CCM, there will be more changes we need to incorporate into these two scripts. Script adjustments are discussed further in the [Adjusting Scripts for Client Setup](#adjusting-scripts-for-client-setup) section below.
+
+## CCM Setup
+
+QDE V1 does not currently include the most up-to-date version of the CCM packages (version 0.3.0, as of this writing). If you have already purchased your Chocolatey for Business (C4B) licenses, you can upgrade by following the [[Central Management Upgrade|CentralManagementSetupUpgrade]] documentation. If you are a trial user, please reach out to your Sales representative for the appropriate packages and procedure for upgrading CCM.
+
+An additional mechanism of security that is highly recommended is the addition of salt additives to the encrypted communication between your endpoints and the CCM Service. As we know, communication from the endpoints to CCM occurs over port 24020, and is secured by SSL certificates. Adding a salt additive on both ends further hashes this encrypted data, and provides another layer of verification. These salt additives should be at least 8 characters, and you will need to provide both additives on the CCM server and endpoint ends when setting up your communication. These are both configuration items that can be set using the `choco config` command, as shown in the example here:
+
+```powershell
+choco config set centralManagementClientCommunicationSaltAdditivePassword 'YourSuperSecureSalt1'
+choco config set centralManagementServerCommunicationSaltAdditivePassword 'YourSuperSecureSalt2'
+```
+
+Further details on configuring CCM, and all available settings, can be found in the [[Central Management Client Setup|CentralManagementSetupClient#config-settings]] documentation.
+
+In the next section, you will need to incorporate both these salt additives into the script that helps you set up your endpoint clients.
+
+## Adjusting Scripts for Client Setup
+
+On-boarding endpoints into CCM will require the running of a `ClientSetup.ps1` script on those endpoints.
+
+**:exclamation: IMPORTANT:** There is a good chance that the current version of the `ClientSetup.ps1` script in the "C:\choco-setup\files" folder on your QDE VM is out-of-date, as we have made numerous updates to it over the past few months. Please use the following command to update your version of this file (to be run in a PowerShell Administrator window):
+
+```powershell
+Invoke-WebRequest -Uri 'https://ch0.co/clientsetup' -OutFile "$env:SystemDrive\choco-setup\files\ClientSetup.ps1"
```

Comment:
While it is good to have short links, always provide what the full link is as well - there is lower trust for redirecting links. Especially surrounding things that are going to run with admin privileges.

Reply:
The reason for the short links throughout is so that we can update the gist if needed. Every time you change a gist, the link to the new raw version changes. If you have another idea, I'm open to it?

Reply:
Have a way for folks to inspect where the short link goes?

I also did not realize the link changed on edits. Private gist?

Reply:
It is currently a private gist, yeah. For lack of a better place to put it publicly. If you have a better idea of where to share, or give users a mechanism to expand, please share?

Reply:
I think likely what we'll end up doing is setting up a repository the QDE could contact to update scripts. Nice thing about QDE is we already know exactly where those files are, so it makes it easy to create a Chocolatey package that can update those files.

Reply:
OK, as long as we give QDE v1 users the ability to update those as well. Also, if that's coming with v2, are we holding off publishing this till them? A BIG concern of mine is, will we have to wholesale revise this again with v2?